std/stdio.d

```diff
@@ -832,21 +832,19 @@ $(D rawRead) always reads in binary mode on Windows.
         return buffer;
     }
 
+    ///
     unittest
     {
-        static import std.file;
+        import std.file;
 
         auto deleteme = testFilename();
```

Comment:
Yes, this should be changed to use `std.file.deleteme`, which I don't know why is not documented but is public and uses in a couple of [places](https://dlang.org/phobos/std_experimental_logger_core.html#.stdThreadLocalLog).